communitylighting.js

```diff
@@ -38,7 +38,26 @@ class CommunityLighting {
         });
 
         Handlebars.registerHelper(CommunityLighting.handlebarsHelpers);
-        
+
+        const baseSoundLoad = Sound.prototype.load;
+
+        Sound.prototype.load = async function load(...args) {
+            console.log(this);
```

Comment:
Whoops, this was a hangover from the code I sent you, we probably don't need to log `this` out